Source/ASButtonNode.mm

```diff
@@ -69,7 +69,7 @@ - (ASImageNode *)imageNode
   ASLockScopeSelf();
   if (!_imageNode) {
     _imageNode = [[ASImageNode alloc] init];
-    [_imageNode setLayerBacked:YES];
+ //   [_imageNode setLayerBacked:YES];
```

Comment:
Why is this commented out?

Reply:
You can't set the tintColor of a layer-backed node without hitting an assertion currently